oak_functions/loader/src/server.rs

```diff
@@ -564,14 +570,23 @@ impl WasmHandler {
             extensions_metadata.insert(name, (ind + EXTENSION_INDEX_OFFSET, signature));
         }
 
-        let request_bytes = request.body;
-        let mut wasm_state = WasmState::new(
+        // keep _endpoints for runtime, give the correspondng switchboard to the WasmHandler
```

Comment:
(nit) We usually start comments with capital letter and end with full stop.

Reply:
Done, thanks. :)